operator_ui/src/pages/Jobs/Definition.tsx

```diff
@@ -59,12 +57,18 @@ const Definition: React.FC<
                 <Divider light className={classes.divider} />
               </Grid>
               <Grid item xs={12}>
-                <PrettyJson
-                  object={jobSpecDefinition({
-                    ...jobSpec,
-                    ...jobSpec.attributes,
-                  })}
-                />
+                {job.type === 'Direct request' && (
```

Comment:
What do you think about keeping 'Direct request' and 'Off-chain reporting' types in enum?

Reply:
I don't think it matters too much 🤷‍♂️ both options give you auto-complete but with strings you need to do less imports.